packages/slice-machine/src/features/environments/actions/getActiveEnvironment.ts

```diff
@@ -15,18 +15,27 @@ type GetActiveEnvironmentReturnType =
       error: unknown;
     };
 
-export async function getActiveEnvironment(): Promise<GetActiveEnvironmentReturnType> {
+export async function getActiveEnvironment(
+  isRetry = false,
```

Comment:
i'd put this in a object, so the caller side is not just `getActiveEnvironment(true)` but something like `getActiveEnvironment({ retried: true })`